### ISSUE TITLE
Purge not found

### ISSUE THREAD

**Issue by wesleywillians** [php-rdkafka/php-rdkafka#300]
* PHP version: 7.3
 * librdkafka version: 
version => 4.0.0
build date => Oct 22 2019 18:19:16
librdkafka version (runtime) => 0.11.6
librdkafka version (build) => 0.11.6.255

 * php-rdkafka version: 4.0.0
 * kafka version: 2.3

Hello guys,

I'm trying to use the recommendation about flushing the producer before the process finishes. However, I'm getting the erro: 

"Call to undefined method RdKafka\Producer::purge()"
Any idea?

My code is like this:

````
public function produce($payload, $key = null): void
    {
        $this->topic->produce(RD_KAFKA_PARTITION_UA, 0, $payload, $key);

        while ($this->producer->getOutQLen() > 0) {
            $this->producer->poll(1);
        }
        $this->producer->purge(RD_KAFKA_PURGE_F_QUEUE);
        $this->producer->flush(1000);
    }
```

**Comment by Steveb-p** [comment]
Probably `purge` does not exist in librdkafka 0.11.

**Comment by nick-zh** [comment]
@Steveb-p is correct, purge is only available from `librdkafka:1.0.0`, it seems this is wrong in the documentation. Apologies  @wesleywillians , i will correct this in the doc

**Comment by wesleywillians** [comment]
> @Steveb-p is correct, purge is only available from `librdkafka:1.0.0`, it seems this is wrong in the documentation. Apologies @wesleywillians , i will correct this in the doc

No problem! Just making sure ;)
